src/changelog/2025/03/device-local-login.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Local Login for Remote Instances
+description: "Local Login option for Remote Instances, powered by Device Agent"
+date: 2025-03-27 12:00:00.0  
+authors: ["stephen-mclaughlin"]
+tags:
+  - changelog
+---
+
+
+We have introduced a new Local Login option for Remote Instances.
+
+This feature allows you to enable or disable local login for you Remote Instance, which is particularly useful in air-gapped environments or times when the FlowFuse platform is unreachable.
```

Comment:
```suggestion
This feature allows you to enable a local login for your Remote Instance, which is particularly useful in air-gapped environments or times when the instance cannot reach the FlowFuse platform.
```

Slight rewording to take emphasis off the platform being unavailable - rather the device's connection being unavailable.